sdcm/tester.py

```diff
@@ -1066,7 +1066,6 @@ def get_cluster_k8s_eks(self):
                 # It should have at least 3 vCPU to be able to hold all the pods
                 disk_size=40,
                 role_arn=self.k8s_cluster.nodegroup_role_arn,
-                provision_type=self.params.get('instance_provision'),
```

Comment:
Neither trello task nor git commit/PR describe the problem you are fixing. Can you please add some details about "why?".

Reply:
It is done in order to use default provision type instead of one that is provided by pipeline